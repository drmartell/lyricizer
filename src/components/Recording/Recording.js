import React from 'react';
import queryString from 'query-string';

export default function Recording() {
  const { artist } = queryString.parse(this.props.location.search);

  return (

    <>
      <p>This is the recording page</p>
    </>
  );
}
