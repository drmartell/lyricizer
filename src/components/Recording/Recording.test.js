import React from 'react';
import { shallow } from 'enzyme';
import Recording from './Recording';

describe('Recording component', () => {
  it('should render Recording', () => {
    const wrapper = shallow(
      <Recording
        match={{ params: { id: 1 } }}
        location={{ search: 'hi' }}
      />);
    expect(wrapper).toMatchSnapshot();
  });
});
