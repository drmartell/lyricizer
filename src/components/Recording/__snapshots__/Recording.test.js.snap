// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recording component should render Recording 1`] = `
<h1>
  loading
</h1>
`;
